src/modules/facility-loan/facility-loan.service.create-amount-amendment-for-loan.test.ts

```diff
@@ -21,6 +22,7 @@ describe('FacilityLoanService', () => {
 
   let acbsAuthenticationService: AcbsAuthenticationService;
   let service: FacilityLoanService;
+  let repaymentScheduleBuilder: RepaymentScheduleBuilder;
```

Comment:
We never set this variable so it's always `undefined` - is that deliberate?

Reply:
no!